llmfoundry/models/layers/ffn.py

```diff
@@ -19,11 +19,21 @@
 log = logging.getLogger(__name__)
 
 
-def _resolve_ffn_hidden_and_exp_ratio(
+def resolve_ffn_hidden_and_exp_ratio(
```

Comment:
suggestion: rename to `resolve_ffn_hidden_size` now that we only return the hidden size

Reply:
Thank you!